core_tic_tac_toe/input_validator.py

```diff
@@ -0,0 +1,29 @@
+import string
+
+
+class InputValidator:
+
+    def is_move_valid(self, input_move, board):
+        if not self.__contains_invalid_character(input_move) and input_move.isdigit():
+            player_input = int(input_move) - 1
```

Comment:
I've now seen `int(input_move) - 1` both in `HumanPlayer` and `InputValidator`. This means that an implementation details of your board is leaking out to other areas. It's worth considering how you can make only the board know about its internal data structure.